### PR TITLE
task/COR-1700-add-docs-for-page-parts

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard28**
## Summary

* Updated Page parts documentation

### CODE REVIEW

**Review by VWSCoronaDashboard18 — DISMISSED** [review]
Looking good!

File: docs/page-parts.md [hunk 1]
```diff
@@ -1,57 +1,84 @@
...
 only be able to edit the links list.
 
-### Rich text
-
-A block of portable text. No configuration fields are needed for this page part.
-
 ### Highlighted items
 
 A list of high lighted items, currently only used on the Actueel pages.
 The document also has `minNumber` and `maxNumber` fields
```
Comment (VWSCoronaDashboard21) [inline docs/page-parts.md:61]: Why is the width of the paragraph different than the other paragraphs?

**Review by VWSCoronaDashboard21 — APPROVED** [review]

File: docs/page-parts.md (hunk 1 above)
Comment (VWSCoronaDashboard28) [reply docs/page-parts.md:61]: Not sure to be honest.

Comment (VWSCoronaDashboard21) [reply docs/page-parts.md]: Not blocking, so I'll resolve this.

**Review by VWSCoronaDashboard21 — APPROVED** [review]
Thank you for your valuable contribution. Your dedication to this project is commendable. Keep up the fantastic work!

**Review by VWSCoronaDashboard27 — APPROVED** [review]

**Pull request merged**